client/src/util/metrics/metrics.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+import { extractProcessVariables } from './processVariables';
+import { getUserTaskMetrics } from './userTasks';
+
+export default async function(file) {
+  const metrics = {
```

Comment:
personally, I wouldn't bootstrap the metrics here to this extend. We already know that we will have more and more metrics in the future, therefore I would keep the initialization as short as possible

```js
let metrics = { }
```

and let the metrics sub-modules do the trick

```js
const userTaskMetrics = await getUserTaskMetrics(file);

metrics = {
  ...metrics,
  tasks: {
    userTask: userTaskMetrics
  }
};
```